src/eCrypt/Options.cs

```diff
@@ -23,6 +23,9 @@ internal class Options
         [Option('t', "target", Required = true, HelpText = "Folder of file that should be encrypted and embedded into generated executable")]
         public string EncryptionTargetPath { get; set; }
 
+        [Option('c', "content-version", Required = false, HelpText = "Version of the content to be encrypted. If provided it will be used for 'file version' property of generated executable")]
```

Comment:
I believe we will need extra validation to ensure the format is matching .net `version` class e.g. `1.1.1.1`